test/transaction.js

```diff
@@ -725,7 +725,39 @@ describe('Transaction', function() {
         fakeStream.emit('error', error);
       });
 
-      it('should retry the transaction', function(done) {
+      it('should retry the transaction for UNKNOWN', function(done) {
```

Comment:
This test needs to mock `isRetryableErrorCode_` and make sure it receives the error code.

Reply:
This test mocks out shouldRetry and shouldRetry now has a test for itself to verify it handles this. So this particular thing seems like testing in the wrong place to me.

Reply:
I just saw why this is desired and have added this unit test.